docs/hosting-a-server-with-northstar/dedicated-server/best-practices.md

```diff
@@ -25,3 +25,11 @@ The current minimum requirements are as follows:
 - 15 Mbps network upload, but 10 is workable, 25 if you want to avoid players getting disconnected when going back to the lobby after a match
 
 **Note:** It is recommended to surpass the listed requirements. Currently the number of available servers covers the daily playerbase more than enough. If you're planning to host public servers for the community it is therefore recommended to either fill a niche (like a gamemode that is so popular that all existing servers are full) or provide a better service than existing hosts (less lag, more stable, etc.).
+
+## Optimization Commands
```

Comment:
Should maybe add a line or so that this is not a strictly necessary thing and should only be applied on a case-by-case basis when found to be useful.

Reply:
Thanks for your remind